browser/brave_ads/ads_tab_helper.cc

```diff
@@ -27,6 +26,15 @@
 
 namespace brave_ads {
 
+namespace {
+
+constexpr char16_t kGetDocumentHTMLScript[] =
```

Comment:
We're injecting this into every page? This seems like a probable perf issue, but what exactly are we using this for anyway? If the purpose of this is to get page content, this is definitely not what we should be doing

Reply:
also this appears to be happening even if ads is disabled?